### ISSUE TITLE
Python TypeError

### ISSUE THREAD

**Issue by paulkeogh** [ConsenSys/mythril#155]
I get the following output for;

myth -xo json -c <bytecode>

```
Traceback (most recent call last):
  File "/home/ubuntu/.local/bin/myth", line 433, in <module>
    'markdown': report.as_markdown() or "The analysis was completed successfully. No issues were detected."
  File "/home/ubuntu/.local/lib/python3.5/site-packages/mythril/analysis/report.py", line 95, in as_markdown
    text += "# Analysis results for " + issue.filename
TypeError: Can't convert 'NoneType' object to str implicitly
```

The bytecode is a random contract taken from the Ropsten chain of;

0x60606040525b33600060006101000a81548173ffffffffffffffffffffffffffffffffffffffff021916908302179055506000600260006101000a81548160ff021916908302179055505b610511806100586000396000f36060604052361561008a576000357c0100000000000000000000000000000000000000000000000000000000900480631ff1292f1461008f578063502a26a1146100ac57806383197ef0146100c95780638da5cb5b146100dd5780639ce6582d1461011b578063b8f7a6651461014e578063e592174114610178578063f832d308146101a25761008a565b610002565b34610002576100aa60048080359060200190919050506101bf565b005b34610002576100c76004808035906020019091905050610238565b005b34610002576100db6004805050610300565b005b34610002576100ef600480505061039a565b604051808273ffffffffffffffffffffffffffffffffffffffff16815260200191505060405180910390f35b346100025761013660048080359060200190919050506103c0565b60405180821515815260200191505060405180910390f35b3461000257610160600480505061041f565b60405180821515815260200191505060405180910390f35b346100025761018a600480505061042d565b60405180821515815260200191505060405180910390f35b34610002576101bd6004808035906020019091905050610449565b005b600060009054906101000a900473ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff163373ffffffffffffffffffffffffffffffffffffffff1614151561021b57610002565b80600260006101000a81548160ff021916908302179055505b5b50565b600060009054906101000a900473ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff163373ffffffffffffffffffffffffffffffffffffffff1614151561029457610002565b60008173ffffffffffffffffffffffffffffffffffffffff1614156102b857610002565b6001600160005060008373ffffffffffffffffffffffffffffffffffffffff16815260200190815260200160002060006101000a81548160ff021916908302179055505b5b50565b600060009054906101000a900473ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff163373ffffffffffffffffffffffffffffffffffffffff1614151561035c57610002565b600060009054906101000a900473ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff16ff5b5b565b600060009054906101000a900473ffffffffffffffffffffffffffffffffffffffff1681565b6000600260009054906101000a900460ff16806104135750600160005060008373ffffffffffffffffffffffffffffffffffffffff16815260200190815260200160002060009054906101000a900460ff165b905061041a565b919050565b60006001905061042a565b90565b6000600260009054906101000a900460ff169050610446565b90565b600060009054906101000a900473ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff163373ffffffffffffffffffffffffffffffffffffffff161415156104a557610002565b60008173ffffffffffffffffffffffffffffffffffffffff1614156104c957610002565b6000600160005060008373ffffffffffffffffffffffffffffffffffffffff16815260200190815260200160002060006101000a81548160ff021916908302179055505b5b5056

**Comment by b-mueller** [comment]
I fixed that particular bug in [b3eb8e6](https://github.com/ConsenSys/mythril/commit/b3eb8e602963b1b5beeab42c84f9b66e6857f928), but Mythril still doesn't like your bytecode. Apparently there's also a stack underflow:

```
$ myth -xo json -c "5550[...]5056"
Traceback (most recent call last):
  File "/Users/bernhardmueller/.pyenv/versions/mythril/bin/myth", line 412, in <module>
    sym = SymExecWrapper(contract, address, max_depth=args.max_depth)
  File "/Users/bernhardmueller/.pyenv/versions/3.6.3/envs/mythril/lib/python3.6/site-packages/mythril/analysis/symbolic.py", line 22, in __init__
    self.laser.sym_exec(address)
  File "/Users/bernhardmueller/.pyenv/versions/3.6.3/envs/mythril/lib/python3.6/site-packages/laser/ethereum/svm.py", line 263, in sym_exec
    node = self._sym_exec(gblState)
  File "/Users/bernhardmueller/.pyenv/versions/3.6.3/envs/mythril/lib/python3.6/site-packages/laser/ethereum/svm.py", line 789, in _sym_exec
    index, value = state.stack.pop(), state.stack.pop()
```

This requires a fix in laser-ethereum.

**Comment by b-mueller** [comment]
Seems to work fine with current master:

```
$ $MYTH -xo json -c "0x60606040525b33600060006101000a81548173ffffffffffffffffffffffffffffffffffffffff021916908302179055506000600260006101000a81548160ff021916908302179055505b610511806100586000396000f36060604052361561008a576000357c0100000000000000000000000000000000000000000000000000000000900480631ff1292f1461008f578063502a26a1146100ac57806383197ef0146100c95780638da5cb5b146100dd5780639ce6582d1461011b578063b8f7a6651461014e578063e592174114610178578063f832d308146101a25761008a565b610002565b34610002576100aa60048080359060200190919050506101bf565b005b34610002576100c76004808035906020019091905050610238565b005b34610002576100db6004805050610300565b005b34610002576100ef600480505061039a565b604051808273ffffffffffffffffffffffffffffffffffffffff16815260200191505060405180910390f35b346100025761013660048080359060200190919050506103c0565b60405180821515815260200191505060405180910390f35b3461000257610160600480505061041f565b60405180821515815260200191505060405180910390f35b346100025761018a600480505061042d565b60405180821515815260200191505060405180910390f35b34610002576101bd6004808035906020019091905050610449565b005b600060009054906101000a900473ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff163373ffffffffffffffffffffffffffffffffffffffff1614151561021b57610002565b80600260006101000a81548160ff021916908302179055505b5b50565b600060009054906101000a900473ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff163373ffffffffffffffffffffffffffffffffffffffff1614151561029457610002565b60008173ffffffffffffffffffffffffffffffffffffffff1614156102b857610002565b6001600160005060008373ffffffffffffffffffffffffffffffffffffffff16815260200190815260200160002060006101000a81548160ff021916908302179055505b5b50565b600060009054906101000a900473ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff163373ffffffffffffffffffffffffffffffffffffffff1614151561035c57610002565b600060009054906101000a900473ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff16ff5b5b565b600060009054906101000a900473ffffffffffffffffffffffffffffffffffffffff1681565b6000600260009054906101000a900460ff16806104135750600160005060008373ffffffffffffffffffffffffffffffffffffffff16815260200190815260200160002060009054906101000a900460ff165b905061041a565b919050565b60006001905061042a565b90565b6000600260009054906101000a900460ff169050610446565b90565b600060009054906101000a900473ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff163373ffffffffffffffffffffffffffffffffffffffff161415156104a557610002565b60008173ffffffffffffffffffffffffffffffffffffffff1614156104c957610002565b6000600160005060008373ffffffffffffffffffffffffffffffffffffffff16815260200190815260200160002060006101000a81548160ff021916908302179055505b5b5056"
^[{"success": true, "error": null, "issues": []}
```
